Tests/Brokerages/Kraken/KrakenBrokerageIntegrationTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using QuantConnect.Brokerages.Kraken;
+using NUnit.Framework;
+using QuantConnect.Interfaces;
+using QuantConnect.Securities;
+using QuantConnect.Configuration;
+using QuantConnect.Orders;
+using System.Reflection;
+using Moq;
+using QuantConnect.Brokerages;
+using RestSharp;
+
+namespace QuantConnect.Tests.Brokerages.Kraken
+{
+    [TestFixture, Ignore("This test requires a configured and active account")]
+    public class KrakenBrokerageIntegrationTests : BrokerageTests
```

Comment:
Awesome stuff -- I'm trusting that these were run and all pass :)

Reply:
I am not sure to be honest. I decided to "live-test" before doing unit tests, I think that way it should be easier.